command-line/keyboard-shortcuts.md

```diff
@@ -53,6 +53,12 @@ You don't have to use keyboard shortcuts for everything on your computer, but af
 
 - [Windows Keyboard Shortcuts Cheat Sheet](https://code.visualstudio.com/shortcuts/keyboard-shortcuts-windows.pdf)
 
+**Mac Utilities (optional):**
+
+* Alfred 3 - The Mac's best launcher just got a whole lot better
```

Comment:
Alfred 3 - Alfred is an award-winning app for macOS which boosts your efficiency with hotkeys, keywords, text expansion and more. Search your Mac and the web, and be more productive with custom actions to control your Mac.
_This could be much more better description for Alfred 3_